src/content/docs/style-guide/word-choice/nerdgraph-doc-guidelines.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: How to write NerdGraph docs
+contentType: page
+template: basicDoc
+---
+
+How to talk about and refer to NerdGraph and related features and functionalities. 
+
+## Style-related guidelines [#nerdgraph-style]
+
+Some important style-related guidelines for NerdGraph: 
+* **NerdGraph.** On first mention, it's okay to phrase it as our "NerdGraph API," especially in situations where it may not be obvious what we're talking about, but generally you should use just "NerdGraph."
+* **NerdGraph is a single API.** NerdGraph is not a set of different APIs. There is sometimes an inclination to refer to NerdGraph as a collection of APIs, which is actually somewhat true because NerdGraph does pull together a lot of our APIs under the hood, but from a customer point of view, we want to refer to it as a single API.
+    * For example, instead of saying something like, "use NerdGraph's entity API" we would say something like "use NerdGraph to view and configure your monitored entities."
+* **GraphQL.** GraphQL is the open-source API format that NerdGraph was built with. It is okay to occasionally highlight this when appropriate because GraphQL is well known in the tech world and some people are already familiar with how it works. 
+* **NerdGraph explorer.** We mainly refer to this as the "NerdGraph explorer" but if it's in a context where it may not be obvious what exactly NerdGraph is, you can refer to the "NerdGraph API explorer." Other tips: 
+    * Avoid using just "explorer" because that conflicts with other explorers (e.g., the REST API explorer, or New Relic Explorer).
+    * We can explain, if we think it's helpful, that the NerdGraph explorer is built with GraphiQL, an open source platform for building GraphQL queries. This may be helpful to explain because both GraphQL and GraphiQL are known in the industry. 
+* **Requests.** There are two main types of GraphQL operations: queries and mutations ([defined here](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology)). We can use these terms to refer to specific examples (for example: "here's a mutation that..."). To make a general reference to multiple NerdGraph requests of different kinds, you can use "requests." 
+* **Fields.** Fields are the elements on an object type that can be queried. For examples of "field" being used in context, see [this NerdGraph tutorial](/docs/nerdgraph-anomaly-detector-configurations-api-tutorial/#update-configuration). The reason fields are an important concept is that with GraphQL-format APIs, users specify exactly the fields they want back (in contract to REST APIs, for example, which can return a default response with fields you don't need). 
+
+## Tips for creating new NerdGraph docs [#creating-nerdgraph-docs]
+
+Tips for adding NerdGraph content to the public docs: 
```

Comment:
@zuluecho9, this tip sounds just like the heading. How about adding an introduction like this: "Since the number of tutorials is growing steadily, please consider the following to keep the content consistent and to keep this section organized:"